lib/openstudio/extension/runner.rb

```diff
@@ -41,13 +41,14 @@ def initialize(dirname = Dir.pwd, bundle_without = [], options = {})
 
         # if the dirname contains a runner.conf file, then use the config file to specify the parameters
 
-        @options = OpenStudio::Extension::RunnerConfig.default_config
+        @options = OpenStudio::Extension::RunnerConfig.default_config(dirname)
```

Comment:
Default config will have the bundle config path or the default .bundle/install already